stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/app.py

```diff
@@ -24,6 +24,11 @@
 settings = ElasticsearchSettings()
 session = Session.create_from_settings(settings)
 
+filter_extension = FilterExtension(client=EsAsyncBaseFiltersClient())
+filter_extension.conformance_classes.append(
+    "http://www.opengis.net/spec/cql2/1.0/conf/advanced-comparison-operators"
```

Comment:
@jamesfisher-gis @philvarner with this link and the other conformance class links, I am getting a not found error?

Reply:
Most OGC conformance class URIs are non-resolvable -- it's just a unique string identifier